### ISSUE TITLE
Direct Midi Render?

### ISSUE THREAD

**Issue by Treata11** [paulrosen/abcjs#979]
Is **abcjs** capable of rendering the _music-notation_ of standard midi files in a direct input?

**Comment by paulrosen** [comment]
No. That would be an awesome feature. I've looked into the problem a little bit and it is not trivial. I think there are some packages that will go from midi to abc, then you could feed that into abcjs.
